### PR TITLE
Universal GSU Kernel

### PR DESCRIPTION
**Pull request by KKyang**

### CODE REVIEW

File: tensilelite/Tensile/Common.py [hunk 1]
```diff
@@ -999,7 +1009,10 @@ def getArchitectureName(gfxName):
     "WorkGroupReduction":          [False],
 
     # 4:2 Structured Sparse A Matrix, 0=Non Sparse, 1=Sparse Matrix A, 2=Sparse Matrix B
-    "Sparse":                      [0, 1, 2]
+    "Sparse":                      [0, 1, 2],
+
+    # Information about user input internal kernel argument support
+    "SupportUserGSU":              [False, True],
```
Comment (jichangjichang) [inline tensilelite/Tensile/Common.py]: Don't add New Kernel Feature just for information purpose.

Comment (KKyang) [reply tensilelite/Tensile/Common.py]: You have to add this since you can't tell if a custom kernel supports GSU. @jichangjichang

Comment (KKyang) [reply tensilelite/Tensile/Common.py]: Or you can just turn it off if it is a custom kernel.

**Review by aazz44ss — APPROVED** [review]

File: tensilelite/Tensile/Common.py (hunk 1 above)
Comment (KKyang) [reply tensilelite/Tensile/Common.py]: Fixed

**Review by jichangjichang — APPROVED** [review]
lgtm

**Pull request merged**